install/0000_30_machine-api-operator_09_deployment.yaml

```diff
@@ -46,9 +46,7 @@ spec:
         runAsNonRoot: true
         runAsUser: 65534
       tolerations:
-      - key: "node-role.kubernetes.io/master"
         operator: "Exists"
-        effect: "NoSchedule"
```

Comment:
CI complains about:
```
error: error validating "STDIN": error validating data: ValidationError(Deployment.spec.template.spec.tolerations): invalid type for io.k8s.api.core.v1.PodSpec.tolerations: got "map", expected "array"; if you choose to ignore these errors, turn validation off with --validate=false
```
It should be at least:
```
tolerations:
- operator: "Exists"
```

Reply:
Ohh yes, thanks for catching it. Fixed it :)